R/produce_at_sds_episodes_tests.R

```diff
@@ -0,0 +1,39 @@
+#' Alarm Telecare and SDS Episodes Tests
+#'
+#' @description Produce the test for the Alarm Telecare and SDS all episodes
+#'
+#' @param data new or old data for testing summary flags
+#' (data is from [get_sc_ch_episodes_path()])
+#'
+#' @return a dataframe with a count of each flag.
+#'
+#' @export
+#' @family social care test functions
+produce_at_sds_episodes_tests <- function(data) {
+  data %>%
+    # create test flags
+    create_demog_test_flags() %>%
+    dplyr::mutate(
+      n_missing_sending_loc = dplyr::if_else(is_missing(.data$sc_send_lca), 1, 0),
+      n_missing_person_id = dplyr::if_else(is_missing(.data$person_id), 1, 0)
+    ) %>%
+    # remove variables that won't be summed
+    dplyr::select(
+      -c(
+        .data$chi,
+        .data$person_id,
+        .data$gender,
+        .data$dob,
+        .data$postcode,
+        .data$record_keydate1,
+        .data$record_keydate2,
+        .data$sc_latest_submission,
+        .data$year,
+        .data$recid,
+        .data$smrtype,
+        .data$sc_send_lca
+      )
```

Comment:
```suggestion
    # keep variables for comparison
    dplyr::select(c(.data$valid_chi:.data$West_Lothian))
```
See function: `produce_source_ch_tests` we can have this as a simlar format and select desirable variables. Check this is righ `valid_chi to west lothian` for selection but this reduces the line of code

Reply:
try testing this code for selecting the last variable e.g. west lothian
dplyr::select(c(.data$valid_chi:dplyr::last_col()))